CMakeLists.txt

```diff
@@ -1438,6 +1438,53 @@ if(WIN32)
   target_link_libraries(mixxx-lib PUBLIC FFTW::FFTW)
 endif()
 
+# libdjinterop (must be enabled explicitly to use)
+option(DJINTEROP "Use djinterop library" OFF)
+if(DJINTEROP)
+  message(STATUS "Enabling djinterop support...")
+  target_compile_definitions(mixxx-lib PUBLIC __DJINTEROP__)
```

Comment:
If possible we should expose *features*, not dependencies:

```suggestion
# Denon Engine library export support
option(DENONENGINE "Support for Denon Engine library export" OFF)
if(DENONENGINE)
  target_compile_definitions(mixxx-lib PUBLIC __DENONENGINE__)
```

Reply:
A perfectly reasonable suggestion - I've pushed another commit to use `ENGINEPRIME` throughout (this is the naming convention we previously agreed to use in class names and expose to the user etc. in PR #2753 ).